docs/modelserving/storage/azure/azure.md

```diff
@@ -0,0 +1,60 @@
+
+# Predict on a InferenceService with saved model on Azure
+
+## Using Public Azure Blobs
```

Comment:
This example is likely outdated, let's rework on it and merge the PR https://github.com/kubeflow/kfserving/pull/1750 onto KServe for the azure blob client upgrade.